docsrc/Makefile

```diff
@@ -4,7 +4,7 @@
 # You can set these variables from the command line.
 SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
-SPHINXPROJ    = spaghetti
+SPHINXPROJ    = PACKAGE_NAME
```

Comment:
This should not be changed?

Reply:
Correct. @rahul799 the `PACKAGE_NAME` variable is used in the [submodule_template](https://github.com/pysal/submodule_template/blob/master/doc/Makefile) in reference to what the package name actually is. Here, this should remain `spaghetti`.

Reply:
Ya, am sorry, actually I downloaded the new Makefile from [submodile template](https://github.com/pysal/submodule_template/blob/master/docsrc/Makefile) & forgot to change the Package_Name here. Thanks @weikang9009, @jGaboardi for pointing this out.